### ISSUE TITLE
CMSPagesFilterBackend for recursive filtering

### ISSUE THREAD

**Issue by jrief** [awesto/django-shop#359]
Besides the classic filtering functionality for `shop.rest.filters.CMSPagesFilterBackend`, there should be an implementation which finds all products and one, which find all products belonging to the specified CMS page and its children.

**Comment by rfleschenberg** [comment]
I have a first implementation of this for our current project. As soon as I have some time, I will merge the code into django-shop.

**Comment by rfleschenberg** [comment]
50% of this issue was fixed in PR #465.

The other 50% are about having a filter backend which finds all products.

It turns out you don't really need a filter backend in this case. You can just do something like this in your CMS app's urls:
```
    url(r'^$', ProductListView.as_view(
        serializer_class=ProductSummarySerializer,
        renderer_classes=(CMSPageRenderer, JSONRenderer, BrowsableAPIRenderer),
    )),
```

@jrief, what do you think? Document this and close the issue?
